webrender/src/renderer.rs

```diff
@@ -4358,8 +4358,14 @@ impl<B: hal::Backend> Renderer<B> {
                 }
             }
         }
-        #[cfg(not(feature = "gl"))]
-        self.device.begin_render_pass(_transit_to_present);
+        #[cfg(not(feature = "gl"))]{
+            if !composite_state.opaque_tiles.is_empty()
+                || !composite_state.clear_tiles.is_empty()
+                || !composite_state.alpha_tiles.is_empty()
+                || partial_present_mode.is_none() {
+                self.device.begin_render_pass(_transit_to_present);
```

Comment:
name shouldn't start with underscore, since it's used

Reply:
It's only used behind `#[cfg(feature = "gl")]` so we need the underscore for gl build to pass without  warning.